dist/api_data.py

```diff
@@ -1345,6 +1352,19 @@ def __cmp__(self, other):
         current transaction.  The value must also not be older than the
         current oldest and stable timestamps.  See
         @ref transaction_timestamps'''),
+    Config('durable_timestamp', '', r'''
```

Comment:
I think there is no `durable_timestamp` support in MongoDB 4.0. This change shouldn't be backported?

Reply:
Removed, thanks Hari!